miden-crypto/Cargo.toml

```diff
@@ -64,7 +64,9 @@ std = [
 [dependencies]
 blake3 = { version = "1.8", default-features = false }
 clap = { version = "4.5", optional = true, features = ["derive"] }
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
Seems like ` k256::elliptic_curve::rand_core` is an old version than the one we have in our crate, this forces us to make some changes to the dependencies